fetch/http-cache/status.html

```diff
@@ -51,7 +51,7 @@
     [
       [200, "OK"],
       [203, "Non-Authoritative Information"],
-      [204, "No Content", ""],
+      [204, "No Content", null],
```

Comment:
Why do we not want to verify that there actually is no content?

Reply:
AIUI in fetch a 204's body should not be an empty string.